test/e2e/cluster_upgrade.go

```diff
@@ -78,6 +88,11 @@ var _ = framework.KubeDescribe("Upgrade [Feature:Upgrade]", func() {
 			cm.Register(func(sem *chaosmonkey.Semaphore) {
 				// Close over f.
 				testServiceRemainsUp(f, sem)
+				testSecretsDuringUpgrade(f, sem)
```

Comment:
As per above comment, you'll need to update the description in the "It()" clause above.